client/html/pool_categories.tpl

```diff
@@ -0,0 +1,30 @@
+<div class='content-wrapper pool-categories'>
+    <form>
+        <h1>Pool categories</h1>
+        <div class="table-wrap">
+            <table>
+                <thead>
+                    <tr>
+                        <th class='name'>Category name</th>
+                        <th class='color'>CSS color</th>
+                        <th class='usages'>Usages</th>
+                    </tr>
+                </thead>
+                <tbody>
+                </tbody>
+            </table>
+        </div>
+
+        <% if (ctx.canCreate) { %>
+            <p><a href class='add'>Add new category</a></p>
+        <% } %>
+
+        <div class='messages'></div>
+
+        <% if (ctx.canCreate || ctx.canEditName || ctx.canEditColor || ctx.canDelete) { %>
```

Comment:
Isn't it a bit elaborate? Above I can only see a button to add a new category

Reply:
It's the same as tag categories, should it be changed also?